index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Zaf's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.cdnfonts.com/css/poppins" rel="stylesheet">
+                
+</head>
+
+<body>
+    <main>    
+        <header>
+            <nav>
+                <a href="#"><img id="logo" src="images/my logo.jpg" alt="My logo"></a>
+                <a href="#"><i class="fa-solid fa-bars"></i></a>
+            </nav>
+        </header>
+
+        <section class="text-container back-image">
+            <h1>I'm Zafron.</br> Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product, feature or website Look through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hesitate to
+                    contact me.</p>
+            <p class="connect"><a href="#">LET'S CONNECT</a></p>
+            <ul>
+                <li><a href="#"><img class="so-icons" src="images/social_icon/Vector.svg" alt="Tweeter"></a></li>
+                <li><a href="#"><img class="so-icons" src="images/social_icon/Linkedin icon.svg" alt="Linkedin"></a></li>
+                <li><a href="#"><img class="so-icons" src="images/social_icon/Vector (1).svg" alt="Gmail"></a></li>
+                <li><a href="#"><img class="so-icons"src="images/social_icon/Vector (2).svg" alt="Github"></a></li>
+                <li><a href="#"><img class="so-icons" src="images/social_icon/Vector (3).svg" alt="Angellist"></a></li>
+            </ul>
```

Comment:
@Kamba56 this was not pointed out because the images are displayed perfectly on my side 🤔 they still are...

Reply:
> @Kamba56 this was not pointed out because the images are displayed perfectly on my side thinking they still are...

Yes, I understand they are displayed perfectly on your end. This is why I added it as an optional requirement.
But if you fix the link, they will be displayed perfectly on all platforms and all PCs.

The only required change is to remove the comments you added after the first review. All other changes are optional but are recommended to implement.

Reply:
@Kamba56 I will do it after finishing the 2nd project. Thanks a lot for pointing out it all.

Reply:
> @Kamba56 I will do it after finishing the 2nd project. Thanks a lot for pointing out it all.

You are welcome 😁🤗